library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -15,30 +15,30 @@
  */
 package com.chuckerteam.chucker.internal.ui.transaction
 
+import android.content.Context
 import android.graphics.Bitmap
+import android.graphics.Color
 import android.os.AsyncTask
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.Menu
 import android.view.MenuInflater
 import android.view.View
-import android.view.ViewGroup
-import android.widget.ImageView
-import android.widget.TextView
 import androidx.appcompat.widget.SearchView
+import androidx.core.content.ContextCompat
 import androidx.core.text.HtmlCompat
 import androidx.fragment.app.Fragment
 import com.chuckerteam.chucker.R
 import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
-import com.chuckerteam.chucker.internal.support.highlight
+import com.chuckerteam.chucker.internal.support.highlightWithDefinedColors
+import kotlinx.android.synthetic.main.chucker_fragment_transaction_payload.*
```

Comment:
Switching to syntatic access views is causing a crash when **navigating back** to the **overview** tab from the request tab:

```
2019-10-24 02:09:04.172 11269-11269/? E/.chucker.sampl: Unknown bits set in runtime_flags: 0x8000
2019-10-24 02:09:14.727 11269-11269/com.chuckerteam.chucker.sample E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.chuckerteam.chucker.sample, PID: 11269
    java.lang.IllegalStateException: binaryData must not be null
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment.setBody(TransactionPayloadFragment.kt:104)
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment.access$setBody(TransactionPayloadFragment.kt:35)
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment$UiLoaderTask.onPostExecute(TransactionPayloadFragment.kt:145)
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment$UiLoaderTask.onPostExecute(TransactionPayloadFragment.kt:123)
        at android.os.AsyncTask.finish(AsyncTask.java:755)
        at android.os.AsyncTask.access$900(AsyncTask.java:192)
        at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:772)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

Reply:
It happens on every single transaction for you?
Can't reproduce this crash somehow :(

Reply:
While investigating this issue I decided that it might be a good idea to use ViewBinding to make sure there are no `nulls` in future. 
https://developer.android.com/topic/libraries/view-binding

But I will do it in next PR to not mix responsibilities. For now, would like to fix this crash.

Reply:
> But I will do it in next PR to not mix responsibilities. For now, would like to fix this crash.

++

> While investigating this issue I decided that it might be a good idea to 
> use ViewBinding to make sure there are no nulls in future.

Regarding this, please open an issue or a PR to start discussing. I have nothing against View Binding per se. On the other hand, I'm generally against introducing other patterns inside the app especially if the impact is minimal. The result is that potential contributors that don't know ViewBinding had to learn it before being able to understand/contribute to the library. Happy to discuss anyway :)

Reply:
@cortinico You might have missed my first question after a few of my other messages.
So, this crash happens on every transaction or only some specific?

Reply:
Yup my bad. Yes for me is happening consistently on every request. Do you need further info to help debugging?

Reply:
I understand what happened there, I can't get it why it happens for you and not while I am testing it.
Could you share the details of device/emulator you used? Which Android version it was?
Want to reproduce it and try to solve it instead of just reverting.

Reply:
![Screenshot from 2019-10-24 23-41-52](https://user-images.githubusercontent.com/3001957/67527699-ed2b5980-f6b7-11e9-82b5-b52a4afa64d6.png)

It's a API 29 emulator with 480x800 mdpi

Reply:
Thanks. And by `navigating back` you mean that you click on tab `Overview` while you are on `Response`, right?

Reply:
It happens when swiping between tabs:
![untitled](https://user-images.githubusercontent.com/3001957/67528324-7beca600-f6b9-11e9-8142-70e6c8b3ac91.gif)

Reply:
Wow, thanks. The thing is that I tried to click on tabs, not swiping and it works) Attaching a gif  as example. Going to dive deep.
![tabs_switch](https://user-images.githubusercontent.com/13467769/67552175-365cc700-f713-11e9-86c9-9d04503fd510.gif)